ocaml/sdk-gen/powershell/autogen/src/XenServerPowerShell.sln

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
I forgot to ask earlier, why do we need a solution file for PS and not for C#? (Strictly speaking neither is essential).

Reply:
so when building from VS22 it kept creating it. I know it's not necessary but I liked the idea that if one downloaded the source it'd just start without generating new files or modifying them.

I think it's been created as a result of switching to NuGet (though it technically isn't a requirement)

Reply:
Generally VS does ask one if they want to save the solution, but it does not create it if one says no. I believe the reason why we never provided solution files is that it is unknown what the users will do with the projects, i.e. modify/build them separately or include them in their own solution in which case the provided solution file is not needed. (Personally, I have my own solution with both C# and PS projects together, but this may not be necessarily what the majority of users do, hence I'm not suggesting shipping it). I think if we want to provide solution files we should do it either for both C# and PS or for none, otherwise it looks untidy, but I'd go for none, because it's another 2 files to maintain.

Reply:
> I think if we want to provide solution files we should do it either for both C# and PS or for none, otherwise it looks untidy, but I'd go for none, because it's another 2 files to maintain.

You make a compelling point, I fully agree. I'll revert the addition of the `sln` ^^